<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiBookmarkManagerDialog</class>
 <widget class="QDialog" name="UiBookmarkManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bookmark Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="importButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Import Bookmarks...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Export Bookmarks...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="foldersGridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="foldersLabel">
         <property name="text">
          <string>Folders</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="newFolderButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="renameFolderButton">
         <property name="text">
          <string>Rename</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="removeFolderButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QTreeView" name="foldersTreeView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget1">
      <layout class="QGridLayout" name="bookmarksGridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="bookmarkLabel">
         <property name="text">
          <string>Bookmarks</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QListView" name="bookmarksListView"/>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="editBookmarkButton">
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="removeBookmarkButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UiBookmarkManagerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UiBookmarkManagerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
